site/docs/plugins/i18n-fluent.md

```diff
@@ -0,0 +1,172 @@
+
+# Internationalization with Fluent
+
+[Fluent](https://projectfluent.org/) is a localization system for natural-sounding translations created and maintained by Mozilla Foundation.
+It has a very powerful and at the same time elegant syntax, that enables even non-developer people to efficiently work with translation files.
+Thanks to the [grammY Fluent integration plugin](https://github.com/grammyjs/fluent) you can easily translate your bot to multiple languages using industry best practices like variable substitution, pluralization, locale negotiation, conditional messages and message reuse.
+
+
+## Install
+
+Install the dependencies:
+
+```shell
+npm install --save @grammyjs/fluent @moebius/fluent
+```
+
+grammY Fluent is built on top of the `@moebius/fluent` package, so you will need to install it as well.
+Also, it is highly advisable to read the documentation of the [original library](https://github.com/the-moebius/fluent#moebiusfluent).
+
+
+## Instantiate Fluent
+
+The next step would be to create a Fluent instance:
+
+```typescript
+import { Fluent } from '@moebius/fluent';
+
+const fluent = new Fluent();
+```
+
+Then, you will need to add at least one translation to the Fluent instance:
+
+```typescript
+await fluent.addTranslation({
+  
+  // Specify locale supported by your translation,
+  // you can specify multiple locales if you want
+  locales: 'en',
+  
+  // You can specufy the translation content directly:
+  source: '{YOUR TRANSLATION FILE CONTENT}',
+
+  // OR you can specify a path to translation file or multiple files:
+  filePath: [
+    `${__dirname}/feature-1/translation.en.ftl`,
+    `${__dirname}/feature-2/translation.en.ftl`
```

Comment:
Does this work cross-platform?

Reply:
To be clear, I am referring to file paths on Windows. I do not mean Deno & Node.

Reply:
All the examples are meant for Node.js and Linux. I don't think that we should complicate the example code to support Windows. The Windows devs should know what to do with this.